packages/vscode-extension/package.json

```diff
@@ -927,6 +931,23 @@
                   }
                 },
                 "additionalProperties": false
+              },
+              {
+                "type": "object",
+                "required": [
+                  "configFile"
```

Comment:
Do we need to make the interface extensible for other tunnelling tools in the future?

Reply:
We can add `type`:  `basis` / `ngrok` in the future. As we discussed in the feature design, to avoid user confusion, there is no type attribute.

Reply:
Make sense. Then I wonder if `configFile` is also required for future tunnelling tools?

Reply:
Now `configFile` is required, in the furture, it will according to the valiue of type.

Reply:
Got. Will there be any compatibility issue?

Reply:
The tasks.json currently generated by this scheme will work on the new version of the Teams Toolkit. (The default value of `type` will be `ngrok`.) 

And the project generated in the newer version of  Teams Toolkit will not work in the older version of  Teams Toolkit.

Reply:
The schema may be contains
```json
                "if": {
                  "properties": {
                    "type": {
                      "contains": {
                        "const": "basis"
                      }
                    }
                  }
                },
                "then": {
                  "not": {
                    "required": [
                      "configFile"
                    ]
                  }
                },
                "else": {
                  "required": [
                    "configFile"
                  ]
                }
```